pkg/providers/aws/provider_postgres.go

```diff
@@ -140,11 +141,38 @@ func (p *PostgresProvider) CreatePostgres(ctx context.Context, pg *v1alpha1.Post
 		errMsg := "failed to create aws session to create rds db instance"
 		return nil, croType.StatusMessage(errMsg), errorUtil.Wrap(err, errMsg)
 	}
+
+	// check is a standalone network is required
+	networkManager := NewNetworkManager(sess, p.Client, logger)
+	isEnabled, err := networkManager.IsEnabled(ctx)
+	if err != nil {
+		errMsg := "failed to check cluster vpc subnets"
+		return nil, croType.StatusMessage(errMsg), errorUtil.Wrap(err, errMsg)
+	}
+	// if a standalone network is required configure the network
+	// (standalone vpc, private subnets, subnet groups, security groups, vpc peering)
+	if isEnabled {
+		// todo handle getting vpc cidr block from _network strat
+		logger.Debug("using temp cidr block")
+		_, tempCIDR, err := net.ParseCIDR("10.0.0.0/26")
+		if err != nil {
+			errMsg := "failed to parse vpc cidr"
+			return nil, croType.StatusMessage(errMsg), errorUtil.Wrap(err, errMsg)
+		}
```

Comment:
Is the idea to cover this in a future PR?

Reply:
yes, i updated the comment and linked the jira where this will be handled